packages/fx-core/src/component/configManager/constant.ts

```diff
@@ -9,6 +9,7 @@ export enum SummaryConstant {
   Succeeded = "(√) Done:",
   Failed = "(×) Error:",
   NotExecuted = "(!) Warning:",
+  Warning = "(!) Warning:",
```

Comment:
It looks like the same as NotExecuted?

Reply:
"NotExecuted" is used elsewhere, and the term "NotExecuted" does not precisely match our validation warning scenario. Therefore, I have added a new enum called "Warning" with the same string value as "NotExecuted".